llama_hub/library.json

```diff
@@ -983,5 +983,14 @@
       "repository",
       "collaborators"
     ]
+  },
+  "BitbucketReader":{
+    "id": "bitbucket_reader",
```

Comment:
nit: the id need to be the same as the folder name "bitbucket" to the tests pass

Reply:
thank you! just changed it